client/js/mini-app/header.js

```diff
@@ -66,6 +66,7 @@ const styles = {
 		display: 'flex',
 		'background-color': '#016454',
 		'border-bottom': '4px solid #3D65F9',
+		'margin-top': '100px',
```

Comment:
This looks good! We need to make the same change for the mobile view too though.

Reply:
Good call, thanks -  merged in master and added a 30px margin-top for mobile.

Reply:
Ok great. If you inspect the header element though, the height is `55px` so the margin should also be `55px` to match

<img width="195" alt="Screen Shot 2019-04-13 at 10 46 43 AM" src="https://user-images.githubusercontent.com/6378435/56083417-88727680-5dd9-11e9-90bf-8e44d110505f.png">